### PR TITLE
Add plugin AKAlias.

### PR DESCRIPTION
**Pull request by irstacks**

### CODE REVIEW

**Comment by irstacks** [comment]
Shellcheck had some good suggestions. But now the robot is wrong. I think.

File: Dev/Shell/aka.1h.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# 
+# Header. 
+echo "= | color=#1853C9"  # ∷ ⚯ ɐ ª
+echo '---'
+
+# Read each line in aka file. 
+while read -r line 
+do 
+	# Strip 'alias ' from line for legibility. 
+	dict="$(echo ${line//alias /})"
```
Comment (keithamus) [inline Dev/Shell/aka.1h.sh]: Its not wrong, this could be `dict="${line//alias /}"`

**Comment by tresni** [comment]
@irstacks If you are okay with what you have you can suppress those warnings.  Just add ` # shellcheck disable=SC2116,SC2086` to the line before.

I will say I don't think shellcheck is wrong on any of those:
* `echo "$(echo $variable)"` is the same as `echo "$variable"`.
* as @keithamus pointed out, for assignment, you can safely just do the variable substitution without need to do the `echo`.

Doesn't make these "wrong", just maybe not optimal :)

**Comment by tresni** [comment]
Not sure why Travis isn't picking up that commit...

**Pull request merged**